pq-crypto/s2n_pq_asm.h

```diff
@@ -15,4 +15,8 @@
 
 #pragma once
 
-#define S2N_ANY_NONPORTABLE_OPTIMIZATIONS_ENABLED   __AVX__ || __AVX2__ || __BMI2__
+#ifdef S2N_ALLOW_NONPORTABLE_OPTIMIZATIONS
```

Comment:
Can you mention this variable in the error message that shows up when users try to use avx optimizations and non-portable code? Then at least people would know how to opt-out if they run into the error and don't care.